c_src/message.h

```diff
@@ -0,0 +1,10 @@
+#ifndef MESSAGE_H
+#define MESSAGE_H
```

Comment:
I recommend the use of `#pragma once` as it's less error prone and less wordy then header guards. We've also standardised around the pragma here as it's more convenient.